api/app/src/Entity/User.php

```diff
@@ -384,24 +334,18 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
      * @var UserResearchResponse|null
      *
      * @JMS\Type("App\Entity\UserResearch\UserResearchResponse")
-     *
      * @JMS\Groups({"user", "satisfaction", "user-research"})
-     *
      * @ORM\OneToMany(targetEntity="App\Entity\UserResearch\UserResearchResponse", mappedBy="user", cascade={"persist"})
      */
     private $userResearchResponse;
 
     /**
      * @var User|null
      *
-     * @ORM\OneToOne(targetEntity="App\Entity\User")
```

Comment:
This was the wrong association type, One-to-many self-referencing - https://www.doctrine-project.org/projects/doctrine-orm/en/3.2/reference/association-mapping.html#one-to-many-self-referencing